.github/workflows/main.yml

```diff
@@ -124,12 +124,19 @@ jobs:
         name:
           - stable
           - beta
+          - stable / wasm
+          - beta / wasm
         include:
-          # This allows us to define targets in the future (e.g. wasm32-unknown-unknown)
           - name: stable
             rust: stable
           - name: beta
             rust: beta
+          - name: stable / wasm
+            rust: stable
+            target: wasm32-unknown-unknown
```

Comment:
TODO: We need to make the tests adhere to this.

Reply:
(So the pass in the CI is not accurate)

Reply:
After looking a bit further into this, I think this is the wrong approach. I believe a wasm target needs to be handled completely separately.